go-controller/pkg/util/pod_annotation.go

```diff
@@ -51,6 +53,9 @@ const (
 	OvnPodAnnotationName = "k8s.ovn.org/pod-networks"
 	// DefNetworkAnnotation is the pod annotation for the cluster-wide default network
 	DefNetworkAnnotation = "v1.multus-cni.io/default-network"
+	// UDNOpenPortsAnnotationName is the pod annotation to open default network pods on UDN pods.
+	// Annotation content has format: "tcp:80,udp:53"
```

Comment:
I think there are 2 advantages:
1. The user is used to writing yaml lists like this and I think it is easier to read.
2. The input is now encoded yaml, so rather than having to add a for loop logic to try to split on csv and read values, you can just unmarshal it to a native data type: var data []map[string]string.